common/app/templates/headerInlineJS/shouldEnhance.scala.js

```diff
@@ -4,8 +4,8 @@
 (function (navigator, window) {
     // Enable manual optin to core functionality/optout of enhancement
     var personPrefersCore = function () {
-        if (window.location.hash === '#core' || window.location.hash === '#gu.prefs.force-core=on') return true;
-        if (window.location.hash === '#nocore' || window.location.hash === '#gu.prefs.force-core=off') return false;
+        if (window.location.hash === '#featuresoff' || window.location.hash === '#core' || window.location.hash === '#gu.prefs.force-core=on') return true;
```

Comment:
would be better to cache calls to `window.location.hash` as this is critical and inlined?